README.md

```diff
@@ -1,2 +1,6399 @@
+                        <strong>Note</strong>: If this parameter is left blank or <code>null</code>, this operation returns all user groups matching all values.
+                    </li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Attributes in Result</td>
+            <td>(Optional) Specify a string array to limit the output by returning only the specified attributes. For example, <code>[&quot;auth-concurrent-override&quot;,&quot;auth-concurrent-value&quot;,&quot;authtimeout&quot;,&quot;company&quot;,&quot;email&quot;,&quot;expire&quot;,&quot;expire-type&quot;,&quot;group-type&quot;,&quot;http-digest-realm&quot;,&quot;id&quot;,&quot;max-accounts&quot;,&quot;member&quot;,&quot;mobile-phone&quot;,&quot;multiple-guest-add&quot;,&quot;name&quot;,&quot;password&quot;,&quot;sms-custom-server&quot;,&quot;sms-server&quot;,&quot;sponsor&quot;,&quot;sso-attribute-value&quot;,&quot;user-id&quot;,&quot;user-name&quot;]&nbsp;</code><br />
+                <strong>Note</strong>: If attributes are not specified, then all attributes will be returned.
+            </td>
+        </tr>
+        <tr>
+            <td>Filter By</td>
+            <td>(Optional) Specify attributes to filter the results according to a set criteria.
+        </tr>
+        <tr>
+            <td>Limit</td>
+            <td>(Optional) Specify the maximum number of results that this operation should return.</td>
+        </tr>
+        <tr>
+            <td>Offset</td>
+            <td>(Optional) Specify the offset value to retrieve a subset of records that starts from the offset value. The offset works with the <em>Limit</em> parameter, which determines how many records to retrieve starting from the offset. Values supported are:
+                <ul>
+                    <li><strong>Default:<code>&quot;0&quot;</code></strong></li>
+                    <li><strong>Minimum:<code>&quot;0&quot;</code></strong></li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Sort By</td>
+            <td>Sort the user groups by the specified field and order the results. You can choose to either sort the results by fields, or can order the results, or both.<br />
+                If you choose 'Field', then you must specify the following parameters:
+                <ul>
+                    <li>Field:&nbsp;Specify the name of the field on which you want to sort the result. For example, <code>auth-concurrent-override, auth-concurrent-value, authtimeout, company, email, expire, expire-type, group-type, http-digest-realm, id, max-accounts, member, mobile-phone, multiple-guest-add, name, password, sms-custom-server, sms-server, sponsor, sso-attribute-value, user-id, user-name</code></li>
+                    <li>Order: Choose the order in which you want to sort the result. You can choose between <strong>Ascending </strong>or <strong>Descending</strong>. By default, this is set to <strong>Ascending</strong>.</li>
```

Comment:
Please replace order description as below to maintain uniformity throughout doc

Order: Select the order in which to sort the results. You can choose from following options:
Ascending (default)
Descending

Reply:
Done